src/pages/Registration/Signup/useSubmit.ts

```diff
@@ -16,7 +18,12 @@ export default function useSubmit() {
   } = useFormContext<FormValues>();
   const navigate = useNavigate();
 
-  async function onSubmit({ email }: FormValues) {
+  async function onSubmit() {
+    const email = user.attributes?.email;
+    if (!email) {
+      throw new Error("Logged in user has no email attribute");
```

Comment:
@ap-justin How is this error possible? Emails are required for all Users. :thinking:

Reply:
amplify-ui `type AmplifyUser` has email defined as `email | undefined` so this check is more of a type guard to remove `undefined` before passing as payload to the endpoint